src/sources/multisrc/wqmangastream/WQMangaStreamScraper.js

```diff
@@ -128,8 +128,9 @@ class WQMangaStreamScraper {
     let chapterText = loadedCheerio('.epcontent').html();
 
     if (sourceId === 53) {
-      let ignore = loadedCheerio('.epcontent > p').next().attr('class');
-      loadedCheerio(`p.${ignore}`).remove();
+      let ignore = loadedCheerio('article > style').text().trim().split(',');
+      ignore.push(...ignore.pop().match(/^\.\w+/));
```

Comment:
```suggestion
      let ignore = loadedCheerio('article > style').text().trim().split(',').pop().match(/^\.\w+/);
```

Reply:
line 131 obtains an array of items from the 'article > style' selector.
The last element of the array contains text which are unneeded for line 133.
So I pop it, match the required pieces and push it back in

The suggested solution only obtains one item (ie: the last item), which is not enough to clean the text